docs/UserGuide.md

```diff
@@ -142,11 +159,106 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
-### Clearing all entries : `clear`
+### Assigning dates to a contact: `contacted`
+
+Manually logs the date of the most recent interaction with the contact.
+
+Format: `contacted INDEX d/DATE n/description`
+* Dates are in `DD-MM-YYYY` format.
+
+Example:
+* `contacted 23 d/28-07-2022 n/Sign Contract`
+
+### Displaying contacted contacts within DAYS: `within`
```

Comment:
`DAYS` should be an Integer, representing number of days from today, not a date, same for `later`, sorry for the confusion.